app/js/views/settings.js

```diff
@@ -26,55 +33,68 @@ export default class SettingsView extends View {
     super(controller);
 
     this.modal  = this.$('gaia-modal');
+    this.header = this.$('gaia-header');
     this.addons = this.$('gaia-list');
 
+    this.closeButton       = this.$('button[data-action="close"]');
+    this.mergeModeButton   = this.$('button[data-action="mergeMode"]');
+    this.doneMergingButton = this.$('button[data-action="doneMerging"]');
+
     this.on('click', 'gaia-button', this._handleClick.bind(this));
     this.on('click', 'button', this._handleClick.bind(this));
+    this.on('click', 'a', this._handleClick.bind(this));
 
-    // this.on('change', 'gaia-switch', this._handleChange.bind(this));
+    this.on('click', 'gaia-checkbox', () => setTimeout(this._handleSelectionChange.bind(this)));
+    this.addons.addEventListener('click', this._handleSelectionChange.bind(this));
   }
 
   template() {
     return settingsViewTemplate;
   }
 
-  setAddons(addons) {
+  setAddons(addons, mergeMode) {
     this.addons.innerHTML = '';
 
     addons.forEach((addon) => {
       var installTime = new Date(addon.installTime);
-      this.addons.innerHTML +=
+
+      this.addons.innerHTML += mergeMode ?
```

Comment:
This is pretty hard to read. It would be better to split this up into fragments that get combined together like this:
```js
var frag = mergeMode ? `lorum ipsum ${addon.foo}` : ``;
var template = `doro set amet ${frag}`;
```